### PR TITLE
Removed unnecessary comments for maryMotherOfTheChurch as feast in Poland and Ukraine (fixes #116).

### LINKED ISSUE #116: `ukraine.js`: Is `maryMotherOfTheChurch` a feast in Ukraine?
@pejulian added this comment in the [`poland.js`](https://github.com/romcal/romcal/blob/romcal-es6/src/calendars/poland.js#L685), so I believe @jarosz should confirm this possible issue. Below is the relevant code.

```bash
// [pejulian] Needs confirmation: In Poland this day is a feast
    {
      "key": "maryMotherOfTheChurch",
      "type": Types[4],
      "moment": ( y => Dates.pentecostSunday( y ).add( 1, 'days'))( year ),
      "data": {
        "prioritized": true,
        "meta": {
          "liturgicalColor": LiturgicalColors.WHITE
        }
      }
    },
```

### PR DESCRIPTION
**Pull request by jarosz**

### CODE REVIEW

File: bin/romcal [hunk 1]
```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env node
-var romcal = require('index.js');
-romcal.calendarFor(process.argv[2]);
+var romcal = require('../index.js');
```
Comment (tukusejssirs) [inline bin/romcal]: @jarosz, what does this change? I think @emagnier should review this.

Comment (jarosz) [reply bin/romcal]: This shouldn't be here :)

**Comment by emagnier** [comment]
Also could you [squash](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git#answer-5201642) your 3 commits together? Here this might not be useful to keep them separated in the git history.

**Comment by jarosz** [comment]
Ready.

**Comment by tukusejssirs** [comment]
@emagnier, you can do it to when merging into `romcal-es6` using either `git merge --squash && git add --all && git commit -m "message" && git push` or using the button in the PRs (sqush and merge), so there should be no need to ask the contributors to squash the commits in their repos/branches. :smiley:

**Comment by emagnier** [comment]
I used Github for years when this button doesn't exists yet. I still have my old habits and don't have the reflex to use these new features.

**Pull request merged**